tests/integration/targets/lookup_hashi_vault/lookup_hashi_vault/tasks/tinyproxy_server.yml

```
@@ -0,0 +1,19 @@
+---
+- name: Install tinyproxy
+  shell: |
+    apt-get update
+    DEBIAN_FRONTEND=noninteractive apt-get --assume-yes install tinyproxy
+  changed_when: false
+
+- name: Configure tinyproxy
+  copy:
+    content: |
+      Port 8001
+      MaxClients 100
+      StartServers 10
```

```suggestion
    content: |
      Port 8001
      MaxClients 100
      StartServers 10
      PidFile /tmp/tinyproxy.pid
```

We should capture the PID somewhere so that a handler at the end of the run can kill the proxy (again keeping n mind this may not always be running in a container).

---

indeed

done